.babelrc

```diff
@@ -15,7 +15,13 @@
   ],
   "env": {
     "test": {
-      "presets": ["@babel/env"]
+      "presets": [
+        ["@babel/env", {
+          "targets": {
+            "node": "current"
```

Comment:
why do we need this?

Reply:
This was added per the [Jest setup docs Babel guidance](https://jestjs.io/docs/en/getting-started#using-babel), specifically to support the use of async/await. Since we're no longer using async/await, this change has been reverted.